lib/aws/google/cached_credentials.rb

```diff
@@ -17,28 +17,45 @@ def initialize(options = {})
 
         @profile = options[:profile] || ENV['AWS_PROFILE'] || ENV['AWS_DEFAULT_PROFILE'] || 'default'
         @session_profile = @profile + '_session'
-        @expiration = Aws.shared_config.expiration(profile: @session_profile) rescue nil
-        @credentials = Aws.shared_config.credentials(profile: @session_profile) rescue nil
+        @expiration = begin
+          Aws.shared_config.expiration(profile: @session_profile)
+        rescue StandardError
+          nil
+        end
+        @credentials = begin
+          Aws.shared_config.credentials(profile: @session_profile)
+        rescue StandardError
+          nil
+        end
```

Comment:
Is this really an improvement over the one-liners? TBH I actually find them easier to read

Reply:
This was prettier's doing, I think it doesn't like naked EOL rescues. I don't either, but I'm indifferent on this change.